package.json

```diff
@@ -1,5 +1,32 @@
 {
+  "name": "CiviHR",
+  "description": "HR software for non-profit",
```

Comment:
Was this description confirmed with the PM?

Reply:
I took it from civihr.org (doesn't really matter in the end, this package won't be public anyway)